find-missing-features.js

```diff
@@ -1,8 +1,21 @@
 'use strict';
 
+import esMain from 'es-main';
+import fs from 'fs-extra';
+import yargs from 'yargs';
+import {hideBin} from 'yargs/helpers';
+
 const BCD_DIR = process.env.BCD_DIR || `../browser-compat-data`;
-const bcd = require(BCD_DIR);
-const tests = require('./tests.json');
+const {default: bcd} = await import(
+  process.env.NODE_ENV === 'test' ?
```

Comment:
How about adding `bcd` and `tests` arguments to `main()` and putting the production bits of this in the `if (esMain(import.meta)` branch?